descholar-ceo/AnnounceIT
# #170865110 User get specific announcement by status api

#### What does this PR ​ do​ ?
As a user, I should be able to create an announcement on the API

#### Description of Task to be completed?
 * Write Unit test for the create-announcement api
 * Write codes to implement the create-announcement api

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* Type these commands : 
    *  `cd AnnounceIT` to go in the root directory of the project
    * `npm install` to install all the packages
    * `touch .env` to create dotenv file which will keep privately sensitive data for you,
    * Open the dotenv created above and copy paste this `PORT=8080` press Enter and copy paste 
     this `TOKEN_KEY=descholar` __you can always modify this data by changing the values of 
     PORT and TOKEN_KEY as you wish,__
    * `npm start` to start the server, sothat the api will be up and running
    * Open [Postman](https://www.getpostman.com/) and 
    create a get request to this url `http://localhost:8080/api/v1/announcements/get-specific-announcement-by-status/pending` as you can see the result shows that you don't have any regiistered announcement yet, [Follow this link to know how to add new announcement](https://github.com/descholar-ceo/AnnounceIT/pull/20)
    * Once you are done with adding new announcement enter again `http://localhost:8080/api/v1/announcements/get-specific-announcement-by-status/pending`, you can specify the status by changing the last word in the url, the case of this example is __pending__

#### Any background context you want to provide?
* Use [Postman](https://www.getpostman.com/), to test it or any other rest-api testing tools available to you

#### What are the relevant pivotal tracker stories?
[170865110](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)


#### Questions:
No Question
